libass/ass_types.h

```diff
@@ -52,32 +99,32 @@ typedef struct ass_library ASS_Library;
 
 /* ASS Style: line */
 typedef struct ass_style {
-    char *Name;
-    char *FontName;
+    char *Name;     //must be a valid non-NULL string pointer; may be an empty string
+    char *FontName; //must be a valid non-NULL string pointer; may be an empty string
     double FontSize;
     uint32_t PrimaryColour;
     uint32_t SecondaryColour;
     uint32_t OutlineColour;
     uint32_t BackColour;
-    int Bold;
-    int Italic;
-    int Underline;
-    int StrikeOut;
-    double ScaleX;
-    double ScaleY;
+    int Bold;      // 0 or 1 (boolean)
+    int Italic;    // 0 or 1 (boolean)
+    int Underline; // 0 or 1 (boolean)
+    int StrikeOut; // 0 or 1 (boolean)
+    double ScaleX; // positive with 1.0 representing 100%
+    double ScaleY; // positive with 1.0 representing 100%
     double Spacing;
     double Angle;
     int BorderStyle;
```

Comment:
it would be useful to document BorderStyle as well, and i could suggest to add a borderstyle enum values or something like constants to be used to set this property

Reply:
The `BorderStyle` values are identical to ASS + libass' ASS extensions. I don't think we need to repeat ASS(-extension) spec/behaviour in each field's doc-comment.